client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -0,0 +1,29 @@
+import { Doc } from "../../../document/types";
+
+export interface Survey {
+  key: SurveyKey;
+  show: (doc: Doc) => boolean;
+  rate: number;
+  src: string;
+  teaser: string;
+  question: string;
+}
+
+enum SurveyKey {
+  CSS_CASCADE_2022 = "css_cascade_2022",
+}
+
+export const SURVEYS: Survey[] = [
```

Comment:
TODO:
- introduce env variable to set startTime and rate
- set stage to start on Saturday
- set prod to start on Monday
- duration same (7 * 24 * 60 * 60)

Reply:
Fixed in 43039c93cafad31f649926372c4ff05d1e3cb909.